modules/installation-launching-installer.adoc

```diff
@@ -453,6 +458,13 @@ ifdef::gcp[]
 you can remove it.
 endif::gcp[]
 
+ifdef::azure-restricted[]
+[NOTE]
+====
+If the cloud provider account that you configured on your host does not have sufficient permissions to deploy the cluster, the installation process stops, and the missing permissions are displayed.
+====
+endif::azure-restricted[]
+
```

Comment:
I had referred an Azure documentation of 4.12 release. This note is not there in 4.13 docs. I think you can remove this note.

Reply:
+1, installer doesn't have such validation yet on azure platform.